nginx/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM nginx:1.15.8-alpine
```

Comment:
I wonder if we could simple make the original public in docker hub and then build our own from that so we get consistency (same base for everyone) and resiliency (we are using a project specific ECR repo).

Reply:
We have https://github.com/wellcometrust/dockerfiles/tree/master/nginx, but it doesn't immediately work here and I don't have time to look at this right now.

This at least gets us more resiliency than before (when a change in the catalogue repo broke the storage service). Are you okay if I merge this as-is?